ExtractorUtils.Test/integration/AuthIntegrationTest.cs

```diff
@@ -50,7 +50,6 @@ public async Task TestClientConfig(CogniteHost host)
 
                 var configList = new List<string>(CDFTester.GetConfig(host));
                 configList.Insert(13, "    implementation: basic");
-                configList.Insert(14, "    token-url: https://login.microsoftonline.com/${BF_TEST_TENANT}/oauth2/v2.0/token");
```

Comment:
Why is this gone, what does this part of the test do now? Since `implementation: basic` does nothing.

Reply:
Since the config this appends to the list of config [here](https://github.com/cognitedata/dotnet-extractor-utils/blob/9f15731a666570967923ccee8871f9f22b4c8023/ExtractorUtils.Test/CDFTester.cs#L102) and there's a check that doesn't allow setting both tenant and token-url, this always throw an error hence the removal.